rails-template.rb

```diff
@@ -0,0 +1,596 @@
+
+  test:
+    <<: *default
+    database: #{app_name}_test
+
+  production:
+    <<: *default
+YML
+
+file "config/routes.rb", <<~RUBY
+  # frozen_string_literal: true
+
+  require "sidekiq/web"
+  require "sidekiq-scheduler/web"
+
+  Rails.application.routes.draw do
+    mount ActionCable.server => "/cable"
+    mount Sidekiq::Web => "/sidekiq" # move to admin once there is authentication
+    get "sitemaps/*path", to: "shimmer/sitemaps#show"
+    resources :files, only: :show, controller: "shimmer/files"
+
+    scope "/(:locale)", locale: /#{I18n.available_locales.join("|")}/ do
+      root "pages#home"
+    end
+  end
+RUBY
+
+file "config/application.rb", <<~RUBY
+  require_relative "boot"
+
+  require "rails"
+  # Pick the frameworks you want:
+  require "active_model/railtie"
+  require "active_job/railtie"
+  require "active_record/railtie"
+  require "active_storage/engine"
+  require "action_controller/railtie"
+  require "action_mailer/railtie"
+  require "action_mailbox/engine"
+  require "action_text/engine"
+  require "action_view/railtie"
+  require "action_cable/engine"
+  # require "rails/test_unit/railtie"
+
+  # Require the gems listed in Gemfile, including any gems
+  # you've limited to :test, :development, or :production.
+  Bundler.require(*Rails.groups)
+
+  module TestApp
+    class Application < Rails::Application
+      # Initialize configuration defaults for originally generated Rails version.
+      config.load_defaults 7.0
+      config.time_zone = "Berlin"
+
+      config.middleware.use Shimmer::CloudflareProxy
+
+      config.action_mailer.default_url_options = {host: ENV["HOST"]} if ENV["HOST"].present?
+      config.active_job.queue_adapter = :sidekiq
+
+      config.assets.paths << Rails.root.join("node_modules")
+    end
+  end
+RUBY
+
+file "config/storage.yml", <<~YML
+  test:
+    service: Disk
+    root: <%= Rails.root.join("tmp/storage") %>
+
+  local:
+    service: Disk
+    root: <%= Rails.root.join("storage") %>
+  s3:
+    service: S3
+    access_key_id: <%= ENV["AWS_ACCESS_KEY_ID"] %>
+    secret_access_key: <%= ENV["AWS_SECRET_ACCESS_KEY"] %>
+    region: <%= ENV["AWS_REGION"] %>
+    bucket: <%= ENV["AWS_BUCKET"] %>
+YML
+
+file ".solargraph.yml", <<~YML
+  plugins:
+    - solargraph-standardrb
+  reporters:
+    - standardrb
+YML
+
+file ".vscode/launch.json", <<~JSON
+  {
+    // Use IntelliSense to learn about possible attributes.
+    // Hover to view descriptions of existing attributes.
+    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+    "version": "0.2.0",
+    "configurations": [
+      {
+        "type": "rdbg",
+        "name": "Rails Server",
+        "request": "launch",
+        "command": "bin/rails",
+        "script": "server",
+        "args": [],
+        "askParameters": false,
+        "useBundler": true
+      }
+    ]
+  }
+JSON
+
+file ".vscode/settings.json", <<~JSON
+  {
+    "editor.formatOnSave": true,
```

Comment:
@JensRavens : With rubocop-rspec, that caused `fit` to be autocorrected to `it`. Just make sure it's does not do something similar with `focus` for Rails tests.